metaplex/js/package.json

```diff
@@ -45,7 +52,15 @@
     "@solana/web3.js": "^1.31.0"
   },
   "devDependencies": {
+    "@metaplex-foundation/amman": "^0.0.9",
+    "@types/jest": "^27.4.0",
+    "esbuild": "^0.14.14",
+    "esbuild-runner": "^2.2.1",
     "eslint": "^8.3.0",
-    "rimraf": "^3.0.2"
+    "jest": "^27.4.7",
```

Comment:
I don't fully understand this sentence 😅 Could you rephrase?

Reply:
`tape` and `spok` are test dependencies we are using instead of `jest` https://github.com/metaplex-foundation/metaplex-program-library/blob/master/token-metadata/js/package.json#L51

Reply:
Done